docs/sidebars.js

```diff
@@ -206,5 +206,9 @@ module.exports = {
       ],
     },
     "plugins/README",
+    {
+        type: "doc",
+        id: "licenses/devspace",
+    },
```

Comment:
Interesting that this was added to the docs. Is that one of the CNCF requirements?

Reply:
Nop but having this available I thought it was a nice addition (also Vcluster has this)

Reply:
Gotcha. Yeah I'm curious if that page will stay since we're redoing the vcluster docs. No problem keeping it, was mostly just curious.